ddterm/shell/sd_journal.js

```diff
@@ -19,7 +19,9 @@
 
 import GLib from 'gi://GLib';
 import Gio from 'gi://Gio';
-
+// BEGIN ESM
+import Gi from 'gi';
+// END ESM
```

Comment:
```suggestion
// BEGIN ESM

import Gi from 'gi';
// END ESM

```

Different types of imports and other things should be separated by an empty line in both ESM and legacy packages

Reply:
Done. I hope I haven't missed any separation. 😅